sunpy/data/test/__init__.py

```diff
@@ -54,9 +68,108 @@ def test_data_filenames():
     excludes = ['*.pyc', '*'+os.path.sep+'__*__', '*.py']
     excludes = r'|'.join([fnmatch.translate(x) for x in excludes]) or r'$.'
 
-    for root, dirs, files in os.walk(rootdir):
+    for root, _, files in os.walk(rootdir):
         files = [Path(root) / f for f in files]
         files = [f for f in files if not re.match(excludes, str(f))]
         test_data_filenames_list.extend(files)
 
     return test_data_filenames_list
+
+
+def bitpix_to_dtype(bitpix):
```

Comment:
Just adding a note here to remind myself to go looking for the equivalent function in astropy and understand why we aren't using that.

Reply:
It's here: https://github.com/astropy/astropy/blob/df3e5bb36120797065c8e326309663ab1a3df788/astropy/io/fits/hdu/base.py#L34. The import would be `from astropy.io.fits import BITPIX2DTYPE`